### PR TITLE
added StatsD instrumentation

### PR DESCRIPTION
**Pull request by binq**
## Summary

- *This work is behind a feature toggle (flipper): NO
- added some StatsD instrumentation to find out why there are so many 403s
- removed retries on sidekiq jobs
- added ENDOFFILE marker to WAVE.txt

## Testing done

- [X] *New code is covered by unit tests*
- There was no StatsD logging before, and now StatsD will record 
- If this work is behind a flipper: NO

## What areas of the site does it impact?
on VYE

## Acceptance criteria

- [X]  I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [X]  No error nor warning in the console.
- [X]  Events are being sent to the appropriate logging solution
- [ ]  Documentation has been updated (link to documentation)
- [X]  No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs
- [X]  Feature/bug has a monitor built into Datadog (if applicable)
- [ ]  If app impacted requires authentication, did you login to a local build and verify all authenticated routes work as expected
- [ ]  I added a screenshot of the developed feature

### CODE REVIEW

**Review by jsimonVA — APPROVED** [review]

File: modules/vye/app/models/vye/user_profile.rb [hunk 1]
```diff
@@ -40,11 +49,44 @@ class Vye::UserProfile < ApplicationRecord
...
+      return
+    end
+
+    unless user.loa3?
+      Rails.logger.error "#{name}: The user(#{user&.user_account&.id}) in session is not LOA3."
+      return
+    end
+
+    if user.ssn.blank?
+      Rails.logger.error "#{name}: The user(#{user&.user_account&.id}) in session does not have an SSN."
```
Comment (rmtolmach) [inline modules/vye/app/models/vye/user_profile.rb:63]: What will `name` evaluate to here? I don't see that variable set anywhere.

Comment (binq) [reply modules/vye/app/models/vye/user_profile.rb:63]: i had it as `self.name` but rubocop removed the `self`. Basically it's the class name. I wish there was a `class_name`

File: modules/vye/app/sidekiq/vye/dawn_dash/activate_bdn.rb [hunk 2]
```diff
@@ -4,7 +4,6 @@ module Vye
   class DawnDash
     class ActivateBdn
       include Sidekiq::Job
-      sidekiq_options retry: 8, unique_for: 12.hours
```
[the remark this answers is not in the document]
Comment (binq) [reply modules/vye/app/sidekiq/vye/dawn_dash/activate_bdn.rb:7]: good god no. will update

File: modules/vye/app/models/vye/user_profile.rb (hunk 1 above)
Comment (rmtolmach) [reply modules/vye/app/models/vye/user_profile.rb:63]: got it! Thanks!

**Review by rmtolmach — APPROVED** [review]
LGTM

**Pull request merged**